src/v3/src/components/WebAuthNSubmitButton/WebAuthNSubmitButton.tsx

```diff
@@ -89,15 +90,14 @@ const WebAuthNSubmit: UISchemaElementComponent<{
       justifyContent={showLoading ? 'center' : undefined}
     >
```

Comment:
Per latest discussion on auth list PR, can you convert all buttons' spacing values to use the closest approximate Odyssey spacing token